### PR TITLE
Day 2 Javascript popup-detail

### PR DESCRIPTION
**Pull request by AliRezaBmeDu**
## Changes

- pre-rendered html content dynamically using javascript.
- adjusted index.html and styles.css files to accommodate changes.

### CODE REVIEW

**Review by wayungi — COMMENTED** [review]
Hi <STUDENT_MENTION>, 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [ ] You have submitted a project, that doesnot have the Usage section in the readme. Kindly add it.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -100,9 +100,9 @@ <h3>My Recent Works</h3>
...
           <hr />
         </div>
-      </section>
+      </section> <!--
       <section class="mp-sec">
-       <!--  <div class="mp-sec-img">
+        <div class="mp-sec-img">
           
           <img src="./assets/images/mp-img.svg" alt="image of multi-post stories" />
           <br />
```
Comment (t-yanick) [inline index.html:108]: - [ ] Kindly remove all commented codes from your codebase. This makes your codebase clean and professional

File: index.html [hunk 2]
```diff
@@ -100,9 +100,9 @@ <h3>My Recent Works</h3>
         <div>
           <hr />
```
Comment (t-yanick) [inline index.html:101]: - [ ] Great job with the desktop version of the popup menu. At the moment, you popup occupies almost the entire window, but from the [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&mode=design) there is a considerable margin at the top, bottom, left and right. Check the screenshots below and adjust the margins of your design.
- **Your design**
image
- **Figma design**
image

File: index.html [hunk 3]
```diff
@@ -100,9 +100,9 @@ <h3>My Recent Works</h3>
...
         </div>
```
Comment (t-yanick) [inline index.html:102]: - [ ] The spacing in the desktop version between your languages and title of recent work does not match the figma design. Check screenshots below and adjust the spacing.
- **Figma design**
image
- **Your design**
image

File: index.html [hunk 4]
```diff
@@ -100,9 +100,9 @@ <h3>My Recent Works</h3>
         <div>
```
Comment (t-yanick) [inline index.html:100]: - [ ] Great job with the mobile version of the popup menu. Here you also have an issue with the margins as they don't match the figma design. Kindly take a look at the screenshots below and fix the margin issues.
- **Your design**
image
- **Figma design**
image

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @paulshaun-kdtk , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark: Collaborative work :+1: 
:heavy_check_mark: Projects added dunamically using JavaScript
:heavy_check_mark: Popup added for both desktop and mobile
:heavy_check_mark: Linters all passing


There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @AliRezaBmeDu, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️
- User clicks the button, the popup with details about the project appears ✔️
- User clicks the close (X) button, the popup disappears ✔️
- Modal popup data is populated with the data from the projects array ✔️
- Good use of Github flow ✔️


## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**